src/app/app.tsx

```diff
@@ -1,13 +1,16 @@
 import React from 'react';
 import { StoreContext, useStoreon } from 'storeon/react';
-import './app.scss';
 import { AppStore } from './app.store';
-import { Report } from './modules/reports/reports.state';
 import { Link, Router, View } from 'react-navi';
 import { compose, mount, route, withView } from 'navi';
-import { Articles } from './components/articles.component';
-import { Reports } from './components/reports.component';
+// Components
```

Comment:
comments for imports are redundant; please remove them

Reply:
Comments for imports removed.